### PR TITLE
Updated EdgeMultiplay Package & Removing Extra Debugging

### PR DESCRIPTION
**Pull request by ah1053**
Recent EdgeMultiplay updates, client and server code reviewed on EdgeMultiplay repos.

Commenting out udp and websocket Debugging on message received since it affects processing time when it's printed often.

### CODE REVIEW

**Review by brucearmstrong — APPROVED** [review]

File: Runtime/Scripts/MobiledgeXUDPClient.cs [hunk 1]
```diff
@@ -25,18 +25,20 @@
 using System.Linq;
 namespace MobiledgeX
 {
-    // MobiledgeXUDPClient is a UDP Client Implementation offered with MobiledgeX Unity Package
-    // MobiledgeXUDPClient concurrency model supports the use of a single queue for
-    // send, and another queue for recieve. MobiledgeXUDPClient here has 1 independent thread
-    // per send or receive direction of communication.
+    /// <summary>
+    ///  MobiledgeXUDPClient is a UDP Client Implementation offered with MobiledgeX Unity Package
```
Comment (lgarner) [inline Runtime/Scripts/MobiledgeXUDPClient.cs:29]: Nit: Period (.)

File: Runtime/Scripts/MobiledgeXUDPClient.cs [hunk 2]
```diff
@@ -120,11 +121,11 @@ public async void RunReceive()
         {
             while (run)
             {
-                Debug.Log("Awaiting Receive...");
+                //Debug.Log("Awaiting Receive...");
                 UdpReceiveResult result = await udpClient.ReceiveAsync();
                 if (result != null && result.Buffer.Length > 0)
                 {
-                    receiveQueue.Enqueue(result.Buffer);
+                    receiveQueue.Enqueue(result.Buffer);
```
Comment (lgarner) [inline Runtime/Scripts/MobiledgeXUDPClient.cs:128]: Space.

File: Runtime/Scripts/MobiledgeXWebSocketClient.cs [hunk 3]
```diff
@@ -212,7 +212,7 @@ public async void RunReceive()
                         byte[] result = response.Values.First().GetBuffer();
                         if (result != null && result.Length > 0)
                         {
-                            Debug.Log("Received: " + result);
+                            // Debug.Log("Received: " + result);
```
Comment (lgarner) [inline Runtime/Scripts/MobiledgeXWebSocketClient.cs:215]: Comments. Change the previous spacing.

**Review by lgarner — APPROVED** [review]
Simple. LGTM.

**Review by franklin-huang-mobiledgex — APPROVED** [review]
Looks good

**Pull request merged**